### PR TITLE
fix: enable Open Payments APIs and HTTP signatures

### PR DESCRIPTION
**Pull request by mkurapov**
### Context
In order for the Postman Open Payment requests to work locally, this change:
- updates ports for on the rafiki-auth package 
- adds local-http-signatures app (basically copied from [here](https://github.com/interledger/rafiki/tree/v1.1.0-alpha/localenv/local-http-signatures)) that signs Open Payments postman requests that require signatures
- adds a (shared) private key to sign all of the requests with on behalf of the rafiki-backend app

I've created a Postman workspace with updated Open Payments requests: 
https://www.postman.com/interledger/workspace/testnet

feel free to fork to test

### CODE REVIEW

File: docker/dev/temp/private-key.pem [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
```
Comment (mkurapov) [inline docker/dev/temp/private-key.pem:1]: just a random test key

File: docker/dev/local-http-signatures/app.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+const Koa = require('koa')
```
Comment (mkurapov) [inline docker/dev/local-http-signatures/app.js:1]: just turned it into a js app to remove all of the TS dependencies and build step since it's so small

Comment (mkurapov) [reply docker/dev/local-http-signatures/app.js:1]: @sabineschaller I don't know if there is a cleaner way to have this here without publishing a docker image

File: docker/dev/.env.dev
Comment (raducristianpopa) [inline docker/dev/.env.dev:1]: We don't need `.env.dev`. We are actually using a local `.env` file for Rapyd's credentials.

File: package.json [hunk 3]
```diff
@@ -14,10 +14,10 @@
     "clean": "pnpm clean:modules && pnpm clean:builds",
     "clean:builds": "find . \\( -name \"dist\" -o -name \".next\" \\) -type d -prune -exec rm -rf '{}' +",
     "clean:modules": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
-    "docker:compose": "docker compose -f ./docker/dev/docker-compose.yml",
+    "docker:compose": "docker compose -f ./docker/dev/docker-compose.yml --env-file ./docker/dev/.env.dev",
```
Comment (raducristianpopa) [inline package.json:17]: We don't need to specify the `.env`.

**Review by raducristianpopa — COMMENTED** [review]
Thank you for helping us with this Max! I can make the changes regarding the environment files.!

**Review by Tymmmy — APPROVED** [review]

**Review by raducristianpopa — APPROVED** [review]
🚀

**Pull request merged**